javascript/react/intro-react/one-way-data-binding.md

```diff
@@ -27,19 +27,17 @@ aspects:
 ---
 ## Content
 
-**React** encourages **unidirectional data flow** by design, making it the canonical example of this approach nowadays.
+**React** encourages **unidirectional data flow** by design, making it the canonical example of this approach nowadays. Along with two-way data binding, these approaches describe the way your application keeps the `view` in sync with the data `model`.
```

Comment:
```suggestion
**React** encourages **unidirectional data flow** by design. Along with two-way data binding, React describes the way your application keeps the `view` in sync with the data `model`.
```

Reply:
We should remove temporal language such as "nowadays" that makes the content more prone to becoming stale.

Reply:
Also, React encourages oneway-data binding, not two.

A child component in React cannot modify data of the parent directly (unless the parent gives it a function to do so somehow).

See [Data down, actions up](https://www.google.com/search?q=Data+down%2C+actions+up&oq=Data+down%2C+actions+up&aqs=chrome..69i57j0l5.130j0j7&sourceid=chrome&ie=UTF-8)